### PR TITLE
fixes & improvements

### PR DESCRIPTION
**Pull request by gi8lino**
- if updating an existing merge request, labels will not be checked
- update exceptions
- fix merge request url

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [v2.1.1](https://github.com/containeroo/helminator/tree/v2.1.1) (202X-XX-XX)
```
Comment (rxbn) [inline CHANGELOG.md]:
```suggestion
## [v2.1.1](https://github.com/containeroo/helminator/tree/v2.1.1) (2020-12-17)
```

**Review by rxbn — APPROVED** [review]
LGTM

File: helminator.py [hunk 2]
```diff
@@ -49,9 +49,9 @@
 templates = Templates(
     branch_name="helminator/{CHART_NAME}",
     merge_request_title="Update {CHART_NAME} chart to {NEW_VERSION}",
-    description="| File | Chart | Change |\n"
+    description="| Name | CHART | File | Change |\n"
```
Comment (rxbn) [inline helminator.py]:
```suggestion
    description="| Name | Chart | File | Change |\n"
```

**Review by rxbn — CHANGES_REQUESTED** [review]
see review

File: helminator.py (hunk 2 above)
Comment (gi8lino) [reply helminator.py]: lgtm

**Review by rxbn — APPROVED** [review]
LGTM

**Pull request merged**